x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/analytics_list/use_table_settings.ts

```diff
@@ -7,6 +7,7 @@
 import { Direction, EuiBasicTableProps, Pagination, PropertySort } from '@elastic/eui';
 import { useCallback, useMemo } from 'react';
 import { ListingPageUrlState } from '../../../../../../../common/types/common';
+import { DataVisualizerIndexBasedAppState } from '../../../../../../../common/types/ml_url_generator';
```

Comment:
Since `useTableSettings()` is now used in other places than just from within the analytics management page, should we move the file to a more general place?